packages/docs/getting-started/using-the-mobile-wallet.md

```diff
@@ -4,11 +4,11 @@ One easy way to experiment with the Alfajores Testnet is to install the Celo Mob
 
 ### Getting Started
 
-Since verifying your phone number costs gas, you have to be invited to the platform to have the funds to pay for the verification process. To do so, please visit [celo.org/build/wallet](https://celo.org/build/wallet) and enter your phone number to be messaged an invitation code. Please note that the Celo Wallet is presently only available for Android, with iOS coming soon.
+You can download the Celo wallet for Android from the [Google Play Store](https://play.google.com/apps/testing/org.celo.mobile.alfajores) or for iOS on [TestFlight](https://testflight.apple.com/join/s212x3Rp). Once you install the app, an account will be created for you and you will be able to verify your phone number. Since verifying your phone number costs gas, you have to fund your account using the [Alfajores faucet](https://celo.org/developers/faucet) to pay for the verification process. 
 
-If you already have an account \(and the corresponding seed phrase\), you can download the Celo wallet from the [play store](https://play.google.com/store/apps/details?id=org.celo.mobile.alfajores) and follow instructions in the app to import your wallet using the seed phrase.
+If you already have an account \(and the corresponding seed phrase\), you can download the Celo wallet for Android from the [Google Play Store](https://play.google.com/apps/testing/org.celo.mobile.alfajores) or for iOS on [TestFlight](https://testflight.apple.com/join/s212x3Rp) and follow instructions in the app to import your wallet using the seed phrase.
 
-If you need more funds, you can always visit [celo.org/build/faucet](https://celo.org/build/faucet) and enter your address to get more Celo Dollars.
+If you need more funds, you can always visit [celo.org/build/faucet](https://celo.org/build/faucet) and enter your address to get more CELO and Celo Dollars (cUSD).
```

Comment:
I thought you can only get CELO. You can also get cUSD?

Reply:
yup!